optuna/study/study.py

```diff
@@ -1141,6 +1141,9 @@ def objective(trial):
     See also:
         :func:`optuna.create_study` is an alias of :func:`optuna.study.create_study`.
 
+    See also:
+        The :ref:`rdb` tutorial provides concrete examples to save and resume optimization using RDB.
```

Comment:
Could you apply the following change to fix the `flake8` error?
```suggestion
        The :ref:`rdb` tutorial provides concrete examples to save and resume optimization using
        RDB.
```